android/app/src/main/java/com/sample/MainActivity.kt

```diff
@@ -74,6 +76,15 @@ class MainActivity : ReactActivity() {
 
             addItemDecoration(DividerItemDecoration(context, DividerItemDecoration.VERTICAL))
         }
+
+        findViewById<TextView>(R.id.runtimeInfo).apply {
+            text = resources.getString(
+                    R.string.runtime_info,
+                    (ReactNativeVersion.VERSION["major"] as Number).toInt(),
```

Comment:
Assuming that `ReactNativeHost.VERSION["key"]` returns you a string, it is sufficient just to call `toInt()` directly on it.